src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Controllers/Api/DemoCustomerController.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using VirtoCommerce.DemoSolutionFeaturesModule.Core.Models.Customer;
+using VirtoCommerce.DemoSolutionFeaturesModule.Core.Services.Customer;
+
+namespace VirtoCommerce.DemoSolutionFeaturesModule.Web.Controllers.Api
+{
+    [Route("api/demo/members")]
+    [ApiController]
+    public class DemoCustomerController : Controller
+    {
+        private readonly IDemoTaggedMemberService _taggedItemService;
+        private readonly IDemoTaggedMemberSearchService _searchService;
+
+        public DemoCustomerController(
+            IDemoTaggedMemberService taggedItemService,
+            IDemoTaggedMemberSearchService searchService
+            )
+        {
+            _taggedItemService = taggedItemService;
+            _searchService = searchService;
+        }
+
+        /// <summary>
+        /// GET: api/demo/members/tagged/{id}
+        /// </summary>
+        [HttpGet]
+        [Route("tagged/{id}")]
+        [ProducesResponseType(typeof(DemoTaggedMember), StatusCodes.Status200OK)]
+        public async Task<ActionResult<DemoTaggedMember>> GetDemoTaggedMember(string id)
+        {
+            var criteria = new DemoTaggedMemberSearchCriteria
+            {
+                MemberId = id,
+                Take = 1
+            };
+
+            var taggedMember = (await _searchService.SearchTaggedMembersAsync(criteria)).Results.FirstOrDefault();
+
+            return Ok(taggedMember);
+        }
+
+
+        /// <summary>
+        /// PUT: api/demo/members/tagged
+        /// </summary>
+        [HttpPut]
```

Comment:
Put should be used only for updating. In this case, would be better to use HttpPost attribute